docs/guide.rst

```diff
@@ -2,4 +2,76 @@
 User Guide
 **********
 
-This user guide provides a walkthrough of the major features in the ``hvpy`` package.
+Welcome to ``hvpy``!
+
+This guide provides a walkthrough of the major features in the ``hvpy`` package.
+``hvpy`` is a python wrapper for the `Helioviewer API <https://api.helioviewer.org/docs/v2/>`_.
+This is the same API that powers `helioviewer.org <https://helioviewer.org>`_.
+
+Basic Usage
+-----------
+Each API endpoint has a function in ``hvpy`` that shares the same name and has the same input parameters.
+For example if you are looking at `getJP2Image <https://api.helioviewer.org/docs/v2/api/api_groups/jpeg2000.html#getjp2image>`_ in the Helioviewer documentation, you would execute this in ``hvpy`` with the following command:
+.. code-block:: Python
+
+    import hvpy
+    from hvpy.datasources import DataSources
+    from datetime import datetime
+
+    jp2 = hvpy.getJP2Image(datetime.now(), DataSources.AIA_94.value)
+
+Managing Return Types
+---------------------
+``hvpy`` will attempt to coerce the result into a Python datatype for you.
+In general, there are 3 types of results you can expect from the API:
+
+1. Raw Data
+2. Strings
+3. JSON
+
+In ``hvpy`` these return types map to:
+
+1. bytearray
+2. str
+3. dict
```

Comment:
```suggestion
1. `bytearray`
2. `str`
3. `dict`
```
This should work.

Reply:
Yep, it makes links to the python docs. Added.